symfony/console/5.3/bin/console

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env php
+<?php
+
+use App\Kernel;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+if (!is_file(dirname(__DIR__).'/vendor/autoload_runtime.php')) {
+    throw new LogicException('Symfony Runtime is missing. Try running "composer require symfony/runtime".');
+}
+
+require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
+
```

Comment:
Related to https://github.com/symfony/recipes/pull/787#pullrequestreview-430223298,
what about adding a comment `@see SymfonyRuntime from the symfony/runtime component`?

In `bin/console` and `public/index.php` file

---

Also related to the docbloc from the component readme:

```
The SymfonyRuntime can resolve and handle many types related to the
symfony/http-foundation and symfony/console components.
Check its source code for more information.
```

Reply:
I don't like adding comments in such files. There are other places to maintain such docs.